### PR TITLE
Add pod_chaos.md

### PR DESCRIPTION
**Pull request by fewdan**
Add pod_chaos.md

### CODE REVIEW

File: doc/pod_chaos.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+* **selector** is used to select pods that are used to inject chaos actions.
+* **scheduler** defines the scheduler rules for the running time of the chaos experiment. For more rule information, see <https://godoc.org/github.com/robfig/cron>.
+
+## Pod Kill Action
+
+> **Note:** 
+> 
+> The detailed description of each field in the configuration template are consistent with that in [Pod Failure](#Pod-Failure-Action).
+
+Below is a sample network partition configuration file:
```
Comment (mapleFU) [inline doc/pod_chaos.md]:
```suggestion
Below is a sample network pod kill configuration file:
```

File: doc/pod_chaos.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+Pod chaos allows you to simulate pod faults, specifically `pod failure` and `pod kill`.
+
+- **Pod Failure** action periodically pauses specific pods.
+
+- **Pod Kill** action kills the specified pod and restarts immediately.
+
+## Pod Failure Action
+
+Below is a sample network partition configuration file:
```
Comment (mapleFU) [inline doc/pod_chaos.md]:
```suggestion
Below is a sample pod failure configuration file:
```

File: doc/pod_chaos.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    cron: "@every 2m"
+```
+
+> For more sample files, see [examples](../examples). You can edit them as needed. 
+
+Description:
+
+* **action** defines the specific chaos action for the pod. In this case, it is pod failure.
+* **mode** defines the mode to run chaos action. Supported mode: `one / all / fixed / fixed-percent / random-max-percent`.
+* **duration** defines the duration for each chaos experiment. In the sample file above, the network partition lasts for 30 seconds.
```
Comment (zhouqiang-cl) [inline doc/pod_chaos.md]: s/network partition/pod failure

File: doc/pod_chaos.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+* **selector** is used to select pods that are used to inject chaos actions.
+* **scheduler** defines the scheduler rules for the running time of the chaos experiment. For more rule information, see <https://godoc.org/github.com/robfig/cron>.
+
+## Pod Kill Action
+
+> **Note:** 
+> 
+> The detailed description of each field in the configuration template are consistent with that in [Pod Failure](#Pod-Failure-Action).
+
+Below is a sample network partition configuration file:
```
Comment (zhouqiang-cl) [inline doc/pod_chaos.md]: s/network partition/pod kill

File: doc/pod_chaos.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
+# Pod Chaos Document
+
+This document helps you to build pod chaos experiments. 
+
+Pod chaos allows you to simulate pod faults, specifically `pod failure` and `pod kill`.
+
+- **Pod Failure** action periodically pauses specific pods.
```
Comment (mahjonp) [inline doc/pod_chaos.md]: What's the affect of the `Pod Failure` action?

File: doc/pod_chaos.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+- **Pod Kill** action kills the specified pod and restarts immediately.
```
Comment (mahjonp) [inline doc/pod_chaos.md]: `restarts immediately` isn't the chaos-mesh behaviour.

File: doc/pod_chaos.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
+# Pod Chaos Document
+
+This document helps you to build pod chaos experiments. 
+
+Pod chaos allows you to simulate pod faults, specifically `pod failure` and `pod kill`.
```
Comment (mahjonp) [inline doc/pod_chaos.md]: can you supplement some details about which case does `Pod failure` suit for use and which case does `Pod Kill` suit for?

Comment (cwen0) [inline doc/pod_chaos.md]: Add a reminder that we do not currently support simulation injection of naked pods, only support some specific pods, such as deployment 、statefulset、daemonset

File: doc/pod_chaos.md (hunk 5 above)
Comment (cwen0) [inline doc/pod_chaos.md]: explain when to use `pod failure` or ` pod kill`

File: doc/pod_chaos.md (hunk 7 above)
Comment (fewdan) [reply doc/pod_chaos.md]: I have added the note.

File: doc/pod_chaos.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
+# Pod Chaos Document
+
+This document helps you to build pod chaos experiments. 
+
+> **note:** 
+> 
+> Chaos mash does not currently support simulation injection of naked pods. And it only supports some specific pods, such as `deployment` 、`statefulset` 、`daemonset`.
```
Comment (mapleFU) [inline doc/pod_chaos.md]:
```suggestion
> Chaos mesh does not currently support simulation injection of naked pods. And it only supports some specific pods, such as `deployment` 、`statefulset` 、`daemonset`. 
```

File: doc/pod_chaos.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+    cron: "@every 2m"
+```
+
+> For more sample files, see [examples](../examples). You can edit them as needed. 
+
+Description:
+
+* **action** defines the specific chaos action for the pod. In this case, it is pod failure.
+* **mode** defines the mode to run chaos action. Supported mode: `one / all / fixed / fixed-percent / random-max-percent`.
+* **duration** defines the duration for each chaos experiment. In the sample file above, the pod failure lasts for 30 seconds.
```
Comment (mapleFU) [inline doc/pod_chaos.md]: Well, I don't think `In the sample file above` is proper here.

**Comment by mahjonp** [comment]
lgtm

**Pull request merged**